extensions/wikia/WikiaMaps/controllers/WikiaMapsMapController.class.php

```diff
@@ -51,13 +49,7 @@ public function createMap() {
 		$this->setData( 'creatorName', $this->wg->User->getName() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "creatorName" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)